package-system/pyside2/Findpyside2.cmake.Windows

```diff
@@ -12,13 +12,40 @@ if (TARGET ${TARGET_WITH_NAMESPACE})
 endif()
 
 # Install the pyside2 site-packages into Python
-add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
+add_library(${TARGET_WITH_NAMESPACE} SHARED IMPORTED GLOBAL)
 
 ly_pip_install_local_package_editable(${CMAKE_CURRENT_LIST_DIR}/pyside2/lib/site-packages pyside2)
 
 # In addition to the libraries, create a Pyside2::Tools library that handles the shiboken and pyside2-lupdate executables that are part o
 # of the pyside2 package
 set(${MY_NAME}_BIN_DIR ${CMAKE_CURRENT_LIST_DIR}/pyside2/bin)
+set(${MY_NAME}_LIB_DIR ${CMAKE_CURRENT_LIST_DIR}/pyside2/lib)
+set(${MY_NAME}_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/pyside2/include)
+
+ly_target_include_system_directories(TARGET ${TARGET_WITH_NAMESPACE}
+    INTERFACE 
+        ${${MY_NAME}_INCLUDE_DIR}
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtConcurrent
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtCore
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtGui
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtNetwork
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtOpenGL
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtOpenGLFunctions
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtPrintSupport
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtSql
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtSvg
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtTest
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtUiTools
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtWidgets
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtWinExtras
+        ${${MY_NAME}_INCLUDE_DIR}/PySide2/QtXml
+)
+
+set_target_properties(${TARGET_WITH_NAMESPACE} PROPERTIES 
+    ${MY_NAME}_SHARE_DIR ${CMAKE_CURRENT_LIST_DIR}/pyside2/share
+    IMPORTED_LOCATION ${SHIBOKEN_SHARED_LIBRARIES} "${${MY_NAME}_LIB_DIR}/libshiboken2.abi3.so"
```

Comment:
the cmake file package-system/pyside2/Findpyside2.cmake.Windows indicates Windows, but the set target properties points to an SO file. Is this correct?

Reply:
Arg, no. Mix up somewhere, thanks.

Reply:
Fixed now.